### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by scottvaughanamido**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,60 +10,24 @@ Amido Stacks targets different cloud providers.
...
 #### Template installation
 
-To install the template locally you'll need to download the [Amido.Stacks.Templates](https://www.nuget.org/packages/Amido.Stacks.Templates/) NuGet package.
-
-You can install it locally to your machine via the command line (the version provided is just an example, please consult the NuGet page for an up-to date version)
+For the latest template version, please consult the Nuget page [Amido.Stacks.CQRS.Events.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/). To install the templates to your machine via the command line:
 
 ```shell
-dotnet new --install Amido.Stacks.Templates::0.0.191
+dotnet new --install Amido.Stacks.CQRS.Templates::0.0.101
```
Comment (underscoreHao) [inline README.md:20]: Since this is the `stacks-dotnet` repo the correct package is `Amido.Stacks.Templates`

File: README.md [hunk 2]
```diff
@@ -10,60 +10,24 @@ Amido Stacks targets different cloud providers.
...
 
-
 ### Template usage
 
 #### Template installation
 
-To install the template locally you'll need to download the [Amido.Stacks.Templates](https://www.nuget.org/packages/Amido.Stacks.Templates/) NuGet package.
-
-You can install it locally to your machine via the command line (the version provided is just an example, please consult the NuGet page for an up-to date version)
+For the latest template version, please consult the Nuget page [Amido.Stacks.CQRS.Events.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/). To install the templates to your machine via the command line:
```
Comment (underscoreHao) [inline README.md:17]: Again, the link should point to - https://www.nuget.org/packages/Amido.Stacks.Templates/ and the name should be `Amido.Stacks.Templates`

File: README.md (hunk 1 above)
Comment (scottvaughanamido) [reply README.md]: Correct template name to "Amido.Stacks.Templates"

File: README.md (hunk 2 above)
Comment (scottvaughanamido) [reply README.md]: Correct template name to "Amido.Stacks.Templates"

File: .template.config/template.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
     "Stacks", 
     "WebApi" 
   ],
-  "name": "Amido Stacks Web Api",
+  "name": "Amido Stacks Web API",
```
Comment (underscoreHao) [inline .template.config/template.json:8]: Perhaps add this same name to the console output

Comment (scottvaughanamido) [reply .template.config/template.json:8]: Changed console output name

**Review by underscoreHao — APPROVED** [review]

**Pull request merged**